cabot_alert_pagerduty/models.py

```diff
@@ -69,11 +73,14 @@ def send_alert(self, service, users, duty_officers):
                 else:
                     client.trigger_incident(service_key,
                                             description,
-                                            incident_key=incident_key)
+                                            incident_key=incident_key,
+                                            details=details,
```

Comment:
will this break unit tests? I see there is one test file with `trigger_incident` assertion.

Reply:
yeah it probably will, please update them.

running them locally is possible, it's just a bit of a pain with the current setup

Reply:
After spending over an hour trying to get testing setup for this project, I am kind of throwing my hands up. It is wanting all sorts of Django configuration and other things just to import cabot in the unit test. 

Is there any documentation I can look at?

Reply:
@Aloshi bump. It may not make sense for us to spend more time setting this up for unit testing. If you environment is already set up, can we pair with you for testing and rollout?